### PR TITLE
Add image for elastic search

### PR DESCRIPTION
**Pull request by GeauxEric**

### CODE REVIEW

**Comment by GeauxEric** [comment]
Would like to introduce the image for elastic search using a single node.
Just fixed formatting.
Let me know what is needed next.

File: src/images/elastic_search.rs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+}
+
+impl ElasticSearch {
+    pub fn with_tag(self, tag: String) -> Self {
+        Self { tag, ..self }
+    }
+    pub fn with_env_vars(self, env_vars: HashMap<String, String>) -> Self {
+        Self { env_vars, ..self }
+    }
```
Comment (thomaseizinger) [inline src/images/elastic_search.rs:31]: `RunnableImage` already has support for adding arbitrary env vars, so we don't really need to expose this.

File: src/images/elastic_search.rs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            env_vars,
+            tag: TAG.to_owned(),
+        }
+    }
+}
+
+impl ElasticSearch {
+    pub fn with_tag(self, tag: String) -> Self {
+        Self { tag, ..self }
+    }
```
Comment (thomaseizinger) [inline src/images/elastic_search.rs:28]: Same here, the tag can be overridden via `RunnableImage`.

**Review by thomaseizinger — CHANGES_REQUESTED** [review]
Two comments, otherwise LGTM.

File: src/images/elastic_search.rs (hunk 1 above)
Comment (GeauxEric) [reply src/images/elastic_search.rs]: Done

File: src/images/elastic_search.rs (hunk 2 above)
Comment (GeauxEric) [reply src/images/elastic_search.rs]: Done

**Comment by GeauxEric** [comment]
ping.
@thomaseizinger 
more comments?

**Review by thomaseizinger — APPROVED** [review]
All good, thanks!

bors r+

**Comment by thomaseizinger** [comment]
> Build failed:
> 
>     * [style](https://github.com/testcontainers/testcontainers-rs/runs/5001244153?check_suite_focus=true)

Seems like the formatter is not yet happy! @GeauxEric

**Comment by GeauxEric** [comment]
> > Build failed:
> > ```
> > * [style](https://github.com/testcontainers/testcontainers-rs/runs/5001244153?check_suite_focus=true)
> > ```
> 
> Seems like the formatter is not yet happy! @GeauxEric

reformatted using `dprint fmt`.

**Pull request merged**